cogs/custom.py

```diff
@@ -23,7 +23,8 @@ async def ping(self, ctx: Context) -> None:
         message = await ctx.send(embed=embed)
         end = time.perf_counter()
         duration = round((end - start) * 1000, 2)
-        embed = Embed(title="Info", description=f":ping_pong: Pong! ({duration}ms)", color=Color.blurple(),)
+        embed = Embed(
+            title="Info", description=f":ping_pong: Pong! ({duration}ms)", color=Color.blurple(),)
```

Comment:
```suggestion
        embed = Embed(title="Info", description=f":ping_pong: Pong! ({duration}ms)", color=Color.blurple())
```
Our line limit is 150, which this line is well within, there is no reason to split it.
Same thing applies to lines: 44-45, 75-76, 89-90